src/imageCache.js

```diff
@@ -52,7 +52,6 @@
             var lastCachedImage = cachedImages[cachedImages.length - 1];
             cacheSizeInBytes -= lastCachedImage.sizeInBytes;
             delete imageCache[lastCachedImage.imageId];
-            lastCachedImage.imagePromise.reject();
```

Comment:
I faced the same issue when working with WADO-RS image loader.
I just changed it to return a `deferred` instead of a `promise`.

WADO-RS (Pull request)
https://github.com/chafey/cornerstoneWADOImageLoader/pull/64/commits/9b9d1a86c299fbbcd605eaedbec91182c431bc24

WADO-URI
https://github.com/chafey/cornerstoneWADOImageLoader/blob/master/src/imageLoaders/loadImage.js#L69